lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
 // lms - views - shopping cart
 // ====================
 
+$enrollment-details-width-difference: $baseline*11.85;
```

Comment:
NIT: wrap all computed variables in parentheses (so in this case: ($baseline*11.85)) 

NIT: It seems odd to have baseline multipliers based on so specific a fraction, so if this can be simplified down that would be great.